tests/test_verbs/test_sample_frac.py

```diff
@@ -0,0 +1,6 @@
+import pytest
```

Comment:
Come to think of it, feel free to just have a file called `test_sample` and add this single test to that file. Having a file with a single test feels *a bit* like overkill.